assets/javascripts/discourse/services/discourse-post-event-api.js

```diff
@@ -0,0 +1,126 @@
+import Service from "@ember/service";
```

Comment:
I tried to have all the reactivity done in this file, so it's easier to know what's going on after a remote call